src/Resources/assets/shop/scss/elasticSearchAutocomplete.scss

```diff
@@ -0,0 +1,92 @@
+.results {
+    margin-top: 0.5rem;
+    position: absolute;
+    left: 50%;
+    -webkit-transform: translateX(-50%);
+    transform: translateX(-50%);
+    min-width: 100%;
+    width: 600px;
```

Comment:
its not working properly, min-width allways will override width, or it will be so buggy